src/ContentBundle/Controller/Backend/ContentController.php

```diff
@@ -203,7 +203,11 @@ public function createAction(Request $request, $siteId = null, $type = 0, $layou
             if (null === $content->getPublishAt()) {
                 $content->setPublishAt(new \DateTime());
             }
-
+            
+            if (!$content->getRoles()) {
+                $content->setRoles($this->container->getParameter('opifer_cms.default_content_access'));
```

Comment:
Setting those "default" roles on the content directly has a drawback; What if the "default" roles change over time, you'll have to update the content items on which those default roles were defined. But, at that time you will have no idea which were default and which were actually intended as `['ROLE_SUPER_ADMIN','ROLE_ADMIN']`.
I'd suggest using default roles in the Voter instead of actually saving them on the content.

Reply:
Also, the current implementation won't effect existing content items, so that would also require a migration.